static/.htaccess

```diff
@@ -22,8 +27,6 @@ RewriteRule ^docs(/(8.0|next))?/components/modeler/desktop-modeler/element-templ
 # Added Self-Managed Modeler redirect
 RewriteRule ^docs/next/self-managed/web-modeler/installation/(.*)$ /docs/next/self-managed/modeler/web-modeler/installation/$1 [R=301,L]
 
-# Renamed the folder and the file name. This page didn't exist before 1.2. 
-RewriteRule ^docs(/(8.0|1.3|1.2|next))?/apis-clients/working-with-apis-clients/?$ /docs$1/apis-tools/working-with-apis-tools/ [R=301,L]
```

Comment:
I _think_ we're fine removing this rule in this PR, but we might find that we need to put it back in for other versions until they're all merged. If we do, we could do one of two things: 

1. Leave the rule as-is, because in this version it will never be hit, because the rules we added above would capture it first, or 
2. Rewrite this rule to require a version (remove the `?` after the list of versions). 

Let's do nothing for now, but just in case other versions start failing after this merges, this might be the issue!